File: Backend/Models/Pedido.cs
using Backend.models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Models
{
    public class Pedido
    {
       
        public int PedidoId { get; set; }
      
        public DateTime Data { get; set; }
       
        public List<Produto> Produtos { get; set; }
      
        public int ClienteId { get; set; }
      
        public Cliente Cliente { get; set; }
     
        public decimal Valor { get; set; }

        public decimal Desconto { get; set; }

        
        public decimal ValorTotal { get; set; }


    }
}

File: Frontend/src/app/Pedido/models/Pedido.ts
import { Cliente } from "src/app/Cliente/model/Cliente";
import { Produto } from "src/app/Produto/models/Produto";

export class Pedido {

  pedidoId: number;
  data: string;
  clienteId: number;
  cliente: Cliente;
  valor: number;
  produtos: Produto[];
  desconto: number;
  valorTotal: number;
  
}
